boards/nucleo144-common/include/board_common.h

```diff
@@ -54,9 +54,11 @@ extern "C" {
 /** @} */
 
 /**
- * @brief   User button
+ * @name   User button
+ * @{
```

Comment:
indeed I reverted it. In other places the button is sometimes defined as a group.